stdlib/os/__init__.pyi

```diff
@@ -341,6 +341,11 @@ class stat_result(structseq[float], tuple[int, int, int, int, int, int, int, flo
         if sys.version_info >= (3, 8):
             @property
             def st_reparse_tag(self) -> int: ...
+        if sys.version_info >= (3, 12):
+            @property
+            def st_birthtime(self) -> int: ...  # time of file creation in seconds
```

Comment:
Hmm, not on my machine! ;)

```pycon
Python 3.13.0a0 (heads/main:d4cea794a7, Sep 21 2023, 13:24:35) [MSC v.1932 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> type(os.fstat(0))
<class 'os.stat_result'>
>>> type(os.fstat(0).st_birthtime)
<class 'float'>
```

```suggestion
            def st_birthtime(self) -> float: ...  # time of file creation in seconds
```

Reply:
I don't have a macOS machine to test with, but it's possible that the existing definition here is also incorrect on this:

https://github.com/python/typeshed/blob/6dfa2852300ed610b93dbf1455fe628d394a1207/stdlib/os/__init__.pyi#L357-L358

Reply:
I can test with a macOS machine.

Reply:
```shell
Python 3.10.10 (main, Mar 28 2023, 07:32:15) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> type(os.fstat(0).st_birthtime)
<class 'float'>
````

Yeah looks like both definitions are wrong :D nice catch